pkg/dcgmexporter/registry.go

```diff
@@ -29,27 +34,65 @@ func NewRegistry() *Registry {
 	}
 }
 
+// Register registers a collector with the registry.
 func (r *Registry) Register(c Collector) {
 	r.collectors = append(r.collectors, c)
 }
 
-func (r *Registry) Gather() (map[Counter][]Metric, error) {
+// SetTransformations sets a slice of Metric transformations
+func (r *Registry) SetTransformations(value []Transform) {
+	r.transformations = value
+}
```

Comment:
I can't find its usage.

Reply:
Good catch. I will remove left overs from the initial implementation.

Reply:
done.